contracts/land/LANDRegistry.sol

```diff
@@ -479,4 +472,87 @@ contract LANDRegistry is Storage, Ownable, FullAssetRegistry, ILANDRegistry {
     assembly { size := extcodesize(addr) }
     return size > 0;
   }
+
+  //
+  // LAND Ping
+  //
+
+  /**
+   * @dev Set date from when the LAND Ping feature should be enabled
+   * @param _gracePeriod Desired amount of time in miliseconds to enable feature
+   */
+  function setGracePeriod(uint _gracePeriod) external onlyDeployer {
+    gracePeriod = _gracePeriod;
+  }
+
+  /**
+   * @dev Set amount of time that should pass for a LAND to be transferred to
+   * a new onwer
+   * @param _deemPeriod Desired amount of time in miliseconds for a LAND to decay
+   */
+  function setDeemPeriod(uint _deemPeriod) external onlyDeployer {
+    deemPeriod = _deemPeriod;
+  }
+
+  /**
+   * @dev Ping an address, keep all that address LAND "alive"
```

Comment:
I would change `Ping an address, keep all that address LAND "alive"` to 
`Ping an address`. 

What do you think?

Reply:
It's fine by me